src/PKSim.Core/Model/PopulationSimulation.cs

```diff
@@ -323,10 +323,7 @@ private IReadOnlyList<DataColumn> drugColumnsFor(string organ, string compartmen
       private DataColumn columnsFor(IndividualResults results, string organ, string compartment, string columnName, string compoundName)
       {
          var column = results.FirstOrDefault(x =>
-               x.QuantityPath.Contains(organ) &&
-               x.QuantityPath.Contains(compartment) &&
-               x.QuantityPath.Contains(columnName) &&
-               x.QuantityPath.Contains(compoundName)
+               isQuantityPathFor(organ, compartment, columnName, compoundName, x)
```

Comment:
This is that bug where if one compound contains the other as substring, you get problems

Reply:
good find